make_atf.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# Copyright (c) 2019 Dell Inc.
+#
+# Written by:  Eric van Gyzen <vangyzen@FreeBSD.org>
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE AUTHOR AND CONTRIBUTORS ``AS IS'' AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED.  IN NO EVENT SHALL THE AUTHOR OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
+# OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+# HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+# SUCH DAMAGE.
+
+tcptestsuite_path=${PREFIX:=/usr/local}/share/tcptestsuite
+packetdrill_path=${LOCALBASE:=/usr/local}/bin/packetdrill
+
+find . -type f -and -name '*.pkt' | sort | \
+    sed -e 's,^\./\(.*\)\.pkt$,\1,' > all_tests
+
+printf '# This file is %sgenerated by make_atf.sh\n' @
+printf '# in the tcptestsuite distribution.\n'
+
+while read Torig; do
+	Tunder=$(basename $Torig | tr - _)
+	cat <<-EOF
+	atf_test_case $Tunder
+	${Tunder}_head() {
+	    atf_set timeout 10
```

Comment:
Question: Does this mean the each test can run only for 10 seconds?

Reply:
Yes.  Sorry, I meant to talk to you about that.  This is certainly ripe for revision.  Do you think we should be able to specify different timeouts for each test?

Reply:
Feel free to find me on `#sparc64` or the FreeBSD slack instance.

Reply:
Some tests need very long timeouts. That is why I used the `-t` argument in the `run-all-*-tests` scripts. That applies only to test groups.

How can I join the FreeBSD slack instance?

Reply:
I don't remember the process.  Start here:

https://freebsd.slack.com/

I _think_ you enter your @FreeBSD.org email address, to which it sends a registration code.

Reply:
I second the fact that the timeout should be made more flexible.

Reply:
This is fixed in commit 3a59d229.